pkl-core/src/test/kotlin/org/pkl/core/LanguageSnippetTestsEngine.kt

```diff
@@ -34,6 +34,17 @@ import org.pkl.core.http.HttpClient
 import org.pkl.core.project.Project
 import org.pkl.core.util.IoUtils
 
+private val excludedOnJvm22Plus = listOf(Regex(".*analyzeInvalidHttpModule.*"))
```

Comment:
This test is excluded on JDK22+ because it seems like the output has changed from the HTTP client. JDK22+ is considered by this changeset to be "experimental" still, so tests are not run there by default.

Reply:
Can you leave a TODO here? Pkl's error messages here should not surface Java's message. We should be able to run this snippet test on all Java versions.